src/types.ts

```diff
@@ -0,0 +1,36 @@
+import * as z from "zod";
```

Comment:
I'm not sure about the import, you are importing all fom zod. Dont you need just `z` function?
`import { z } from "zod";`